pkg/management/component.go

```diff
@@ -1172,7 +1172,7 @@ func (c *component) ExecuteActionsEmail(ctx context.Context, realmName string, u
 	values := append(paramKV, "required_actions", listActions)
 
 	details := map[string]string{}
-	for i := 0; i+1 < len(details); i += 2 {
+	for i := 0; i+1 < len(values); i += 2 {
```

Comment:
Could we have a test that would have detected this mistake ?

Reply:
Yes, but not easily, we usually do not check the values of audit events